terragrunt/aws/hosted_zone/outputs.tf

```
@@ -0,0 +1,3 @@
+output "hosted_zone_id" {
+  value = aws_route53_zone.url_shortener.zone_id
```

Could you add a `description` attribute to this.  It could be something as simple as:
```suggestion
  description = "Route53 hosted zone ID that will hold our DNS records."
  value       = aws_route53_zone.url_shortener.zone_id
```

---

Done!